docs/creating_integration_manifest.md

```diff
@@ -174,13 +174,25 @@ The following example will match Nespresso Prodigio machines:
 }
 ```
 
-The following example will match a service uuid used for SwitchBot devices:
+The following example will match a service data uuid used for SwitchBot bot and curtain devices:
```

Comment:
We've switched the order of the legend compared with code examples.

Reply:
Thanks for catching that 👍